forgae-shape-vue-command.md

```diff
@@ -125,95 +125,114 @@ As a result, we can see a table with the most important details. For example:
 
-```javascript=
-async getClient() {
-    try {
-        const networkId = settings.networkId;
 
-        const clientNative = await Ae.compose({
-            props: {
-                url: settings.url,
-                internalUrl: settings.internalUrl,
-                compilerUrl: compilerUrl
-            }
-        })({ nativeMode: true, networkId })
+## Important considerations
 
-        const account = { secretKey: settings.account.priv, publicKey: settings.account.pub }
 
-        await clientNative.setKeypair(account)
+### Identity provider
 
-        this.client = clientNative
+#### Get client
+The application flow starts with getting the client. The code placed in the ```created``` lifecycle hook takes care of this.
 
-        this.$store.dispatch('setAccount', this.client);
-        this.accountBalance();
-        this.contractInstance = await this.client.getContractInstance(contractDetails.contractSource, { contractAddress: contractDetails.contractAddress });
-    } catch (err) {
-        console.log(err);
-    }
+```javascript=
+async created() {
+    this.client = await Wallet({
+        url: this.url,
+        internalUrl: this.internalUrl,
+        compilerUrl: this.compilerUrl,
+        accounts: [MemoryAccount({ keypair: { secretKey: this.priv, publicKey: this.pub } })],
+        address: this.pub,
+        onTx: this.confirmDialog,
+        onChain: this.confirmDialog,
+        onAccount: this.confirmDialog,
+        onContract: this.confirmDialog
+    })
+
+    if (!this.runningInFrame) this.$refs.aepp.src = this.aeppUrl
+    else window.parent.postMessage({ jsonrpc: '2.0', method: 'ready' }, '*')
+
+    this.height = await this.client.height()
+    this.balance = await this.client.balance(this.pub).catch(() => 0)
 }
 ```
 
-### Calling a contract function
+### ToDo aepp
+
+#### Identity 
+The loaded ToDo aepp waits for the ‘parent’ identity provider Aepp to provide the connected client, using the Aepp approach:
+```javascript=
+this.client = await Aepp({
+    parent: this.runningInFrame ? window.parent : await this.getReverseWindow()
+});
+```
+
+#### Configurations
+The configuration file about contract details.
+
+```
+../vue-shape/aepp-forgae-shape-vue/src/contractDetails.js
+```
+We should provide the deployed contract address and the source code of the contract.
+
+
+#### Calling a contract function
 
 In order to interact with the contract functions, we will use two functions:
 
-- callContract - calling a contract function that changes the state of the contract (has ```stateful``` modifier);
+- call - calling a contract function that changes the state of the contract (has ```stateful``` modifier);
 - callStatic - calling a contract function that just read from the contract, without changing the state;
 
 ```javascript=
-async onCallStatic(func, args, returnType) {
-    if (func && args && returnType) {
+async call(funcName, funcArgs) {
```

Comment:
- why overcomplicate this by wrapping in multiple functions?
 - why not use .catch instead of try catch?
 - why document this name and args check? (not needed if not extra wrapping)